lib/collabland/collablandClient.ts

```
@@ -143,16 +143,10 @@ export async function canJoinSpaceViaDiscord({
 }
 
 export async function getGuildRoles(discordServerId: string) {
-  try {
-    const allRoles = await fetch<ExternalRole[]>(`${COLLABLAND_API_URL}/discord/${discordServerId}/roles`, {
-      headers: getHeaders()
-    });
-
-    // filter out irrelevant roles
-    return allRoles.filter((role) => role.name !== '@everyone' && !role.managed);
-  } catch (error) {
-    log.error(`Failed to fetch roles for server ${discordServerId}`, { error, apiDomain: COLLABLAND_API_URL });
+  const allRoles = await fetch<ExternalRole[]>(`${COLLABLAND_API_URL}/discord/${discordServerId}/roles`, {
```

removed the try/catch here - this is just wrapping the fetch call, i think it's better to let callers of this determine whether to swallow the error or propagate it. It might be a missing API key, it could also be that collab land's API is down.

---

+1, good devX improvement 👍